@navikt/core/react/src/tag/Tag.tsx

```diff
@@ -54,23 +60,31 @@ export interface TagProps extends HTMLAttributes<HTMLSpanElement> {
  * ```
  */
 export const Tag = forwardRef<HTMLSpanElement, TagProps>(
-  ({ children, className, variant, size = "medium", icon, ...rest }, ref) => (
-    <BodyShort
-      {...rest}
-      ref={ref}
-      as="span"
-      size={size === "medium" ? "medium" : "small"}
-      className={cl(
-        "navds-tag",
-        className,
-        `navds-tag--${variant}`,
-        `navds-tag--${size}`,
-      )}
-    >
-      {icon && <span className="navds-tag__icon--left">{icon}</span>}
-      {children}
-    </BodyShort>
-  ),
+  ({ children, className, variant, size = "medium", icon, ...rest }, ref) => {
+    const filledVariant = variant?.endsWith("-filled") && "strong";
+    const moderateVariant = variant?.endsWith("-moderate") && "moderate";
+    const color = variant?.replace("-filled", "").replace("-moderate", "");
+
+    return (
+      <BodyShort
+        {...rest}
+        ref={ref}
+        as="span"
+        size={size === "medium" ? "medium" : "small"}
+        className={cl(
+          "navds-tag",
+          className,
+          `navds-tag--${variant}`,
+          `navds-tag--${size}`,
+          `navds-tag--${filledVariant || moderateVariant || "outline"}`,
+        )}
+        data-color={color}
```

Comment:
Think my tran of though was:
- Color is never something you will change in your own CSS here. You will for example never write `.navds-tag--success { color: red }` or any variations of that.
- On the other side you might change it so that "when moderate, change som CSS so that the border is now 2px instead of 1px".

But in tbh, in its mostly arbitrary in this case. What do you find make the most sense here, both className or both data-attb?

Reply:
Not sure. Your trail of thought might make sense. 🤔🤔🤔

A benefit of className is that it might be easier to find usage (overrides) by searching in GitHub. The "navds"-prefix is unique to Aksel, while "data-something" is potentially not. But maybe we don't need to be able to find overrides with data-atributes? 🤔

Does it make sense to only use data-attributes for states like open/closed etc.?

Reply:
> Does it make sense to only use data-attributes for states like open/closed etc.?

Can run with that for now 👍 (Holy misspellings in my previous message lol)